Source/WebCore/rendering/TextDecorationPainter.h

```diff
@@ -67,7 +67,7 @@ class TextDecorationPainter {
         float clippingOffset { 0.f };
         WavyStrokeParameters wavyStrokeParameters;
     };
-    void paintBackgroundDecorations(const TextRun&, const BackgroundDecorationGeometry&, OptionSet<TextDecorationLine>, const Styles&);
+    void paintBackgroundDecorations(const TextRun&, const BackgroundDecorationGeometry&, OptionSet<TextDecorationLine>, const Styles&, const RenderStyle& style);
```

Comment:
same as above (expanding/getting rid of)

Reply:
Nit: You don't need a name for the `const RenderStyle&` argument.